test/verify/check-system-info

```diff
@@ -696,12 +696,14 @@ password=foobar
         self.assertIn("123-nice-id", b.attr(".system-health-insights a", "href"))
         b.logout()
 
+    @testlib.nondestructive
     def testOverview(self):
         m = self.machine
         b = self.browser
 
         # packagekit often eats a lot of CPU; silence it to not screw up the "system is idle" test
         m.execute("systemctl mask packagekit")
+        self.addCleanup(m.execute, "systemctl unmask packagekit")
```

Comment:
Woops :sweat_smile:

Reply:
It was destructive before, so not a bug. But now we need to clean up after ourselves :grin: